saleor/graphql/discount/sorters.py

```diff
@@ -11,7 +11,7 @@ class SaleSortField(BaseEnum):
     START_DATE = ["start_date", "name", "pk"]
     END_DATE = ["end_date", "name", "pk"]
     VALUE = ["value", "name", "pk"]
-    TYPE = ["type", "name", "pk"]
+    TYPE = ["rules__reward_value_type", "name", "pk"]
```

Comment:
It should be done in similar wat as sorting by `value`. We should have a dedicated function for that